### ISSUE TITLE
padding

### ISSUE THREAD

**Issue by SKOTSKOT** [mackenly/simple-icons-stream-deck#10]
is there a way to add padding to icons with the script ?

**Comment by mackenly** [comment]
@SKOTSKOT Can you expand on the use case for this?

**Comment by SKOTSKOT** [comment]
I just think that it would look better if the icons have a bit of padding.

**Comment by mackenly** [comment]
@SKOTSKOT Makes sense. Some of the ones are definitely very cramped, where the stream deck buttons kinda cut off the edges. I'll explore it. Thanks!
